<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>937</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>List Employees</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Employees</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>List Vendors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Vendors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>List Customers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Customers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pay Employee</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>480</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Payroll</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>540</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Inventory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>600</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Invoice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>640</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Invoice History</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>700</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create PO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>740</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PO History</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>800</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Balance Sheet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>840</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Income Statement</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>-10</y>
      <width>751</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; font-style:italic; color:#55aa00;&quot;&gt;Financial Management System:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; font-style:italic; color:#55aa00;&quot;&gt;Main Menu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>340</y>
      <width>392</width>
      <height>236</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_Menu">
    <property name="title">
     <string>Main Menu</string>
    </property>
   </widget>
   <addaction name="menuMain_Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
